packages/mobile/src/components/composer-input/ComposerInput.tsx

```diff
@@ -79,13 +93,15 @@ export const ComposerInput = (props: ComposerInputProps) => {
     messageId,
     placeholder,
     presetMessage,
+    entityId,
     extraOffset = 0
   } = props
   const [value, setValue] = useState(presetMessage ?? '')
   const [selection, setSelection] = useState<{ start: number; end: number }>()
   const { primary, neutralLight7 } = useThemeColors()
   const hasLength = value.length > 0
   const messageIdRef = useRef(messageId)
+  const { data: track } = useGetTrackById({ id: entityId ?? -1 })
```

Comment:
Does audius-query ignore `-1` ids? i remember there was an issue with -1 being passed somewhere else that we were trying to find the source of

Reply:
yea it checks for -1 and returns null in that case